common/devdoc/gballoc_ll_win32heap.md

```diff
@@ -0,0 +1,107 @@
+# gballoc_ll_win32heap requirements
+================
+
+## Overview
+
+gballoc_ll_win32heap is a module that delegates all call of its APIs to the ones from windows heap functions.
+
+## References
+[Heap Functions](https://docs.microsoft.com/en-us/windows/win32/memory/heap-functions)
+
+
+## Exposed API
+
+```c
+    MOCKABLE_FUNCTION(, int, gballoc_ll_init, void*, params);
+    MOCKABLE_FUNCTION(, void, gballoc_ll_deinit);
+
+    MOCKABLE_FUNCTION(, void*, gballoc_ll_malloc, size_t, size);
+    MOCKABLE_FUNCTION(, void, gballoc_ll_free, void*, ptr);
+    MOCKABLE_FUNCTION(, void*, gballoc_ll_calloc, size_t, nmemb, size_t, size);
+    MOCKABLE_FUNCTION(, void*, gballoc_ll_realloc, void*, ptr, size_t, size);
+```
+
+### gballoc_ll_init
+```c
+MOCKABLE_FUNCTION(, int, gballoc_ll_init, void*, params);
+```
+
+`gballoc_ll_init` globally initializes the module. `params` exists as a placeholder and is ignored.
+
+**SRS_GBALLOC_LL_WIN32HEAP_02_001: [** `gballoc_ll_init` shall call `CreateHeap(0,0,0)`. **]**

```

Comment:
>CreateHeap [](start = 69, length = 10)

I think this is `HeapCreate`

Reply:
Correct. Replaced here and everywhere

---
In reply to: [457682925](https://github.com/Azure/azure-c-pal/pull/11#discussion_r457682925) [](ancestors = 457682925)